src/lib/store/actions.js

```diff
@@ -34,9 +34,12 @@ export function fetchTxInfo () {
     dispatch(update('txGasEstimate', gasEstimate));
     dispatch(update('txGasPrice', gasPrice));
     dispatch(update('txBalance', balance));
-    dispatch(update('txValue', params.value));
-  }
 
+    dispatch(update('txValue', params.value || 0));
+    dispatch(update('txURL', first.url || 'unknown'));
+
+    dispatch(update('txTo', params.to || 'N/A'));
```

Comment:
As other comment, even prior to this pull request - when tx's have 0 value (very common) then this error is logged all the time:

> Uncaught (in promise) Error: Given action "UPDATE", reducer "value" returned undefined. To ignore an action, you must explicitly return the previous state. If you want this reducer to hold no value, you can return null instead of undefined.

Reply:
I have left it escaped to 0 to avoid the errors